docs/docs/reference/project-files/dashboards.md

```diff
@@ -6,32 +6,38 @@ sidebar_position: 30
 
 In your Rill project directory, create a `<dashboard_name>.yaml` file in the `dashboards` directory. Rill will ingest the dashboard definition next time you run `rill start`.
 
+:::info Did you know?
+
+Files that are *nested at any level* under your `dashboards` directory will be assumed to be metric definitions.
+
+:::
+
 ## Properties
 
-_**`model`**_ — the model name powering the dashboard with no path _(required)_
+**`model`** — the model name powering the dashboard with no path _(required)_
```

Comment:
We also support `table:` instead of `model:` (they are aliases under the hood). The motivation for `table:` is that a dashboard can also run a) directly on a source, b) on a pre-existing table (as is the case with our current Druid-backed projects).

This PR might not be the place to discuss it, but we might also consider standardizing on `table:` for all cases?

Reply:
That's a great callout and at least I'll mention it in the docs until we've decided whether we want to standardize on one over the other